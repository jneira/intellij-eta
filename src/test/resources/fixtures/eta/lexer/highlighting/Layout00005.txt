ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00005')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('getOAuthTokens')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n  ')
ITlet ('let')
WHITE_SPACE (' ')
ITvarid ('r')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('u')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('initReq')
WHITE_SPACE ('\n                ')
ITocurly ('{')
WHITE_SPACE (' ')
ITvarid ('method')
ITequal ('=')
ITstring ('"')
ITstring ('POST')
ITstring ('"')
WHITE_SPACE ('\n                ')
ITcomma (',')
WHITE_SPACE (' ')
ITvarid ('requestHeaders')
ITequal ('=')
ITobrack ('[')
ITcbrack (']')
WHITE_SPACE ('\n                ')
ITccurly ('}')
WHITE_SPACE ('\n  ')
ITvarid ('return')
WHITE_SPACE (' ')
ITvarid ('r')
WHITE_SPACE ('\n')
