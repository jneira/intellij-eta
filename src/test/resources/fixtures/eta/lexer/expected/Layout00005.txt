ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Layout0000')
WHITE_SPACE ('5')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('getOAuthTokens')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n  ')
ITvocurly ('')
WHITE_SPACE ('')
ITlet ('let')
WHITE_SPACE (' ')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('r')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('u')
WHITE_SPACE (' ')
ITvarsym ('$')
WHITE_SPACE (' ')
ITvarid ('initReq')
WHITE_SPACE ('\n                ')
ITocurly ('{')
WHITE_SPACE (' ')
ITvarid ('method')
WHITE_SPACE ('')
ITequal ('=')
WHITE_SPACE ('')
ITstring ('"POST"')
WHITE_SPACE ('\n                ')
ITcomma (',')
WHITE_SPACE (' ')
ITvarid ('requestHeaders')
WHITE_SPACE ('')
ITequal ('=')
WHITE_SPACE ('')
ITobrack ('[')
WHITE_SPACE ('')
ITcbrack (']')
WHITE_SPACE ('\n                ')
ITccurly ('}')
WHITE_SPACE ('\n  ')
ITvccurly ('')
WHITE_SPACE ('')
ITsemi ('')
WHITE_SPACE ('')
ITvarid ('return')
WHITE_SPACE (' ')
ITvarid ('r')
WHITE_SPACE ('\n')
ITvccurly ('')
WHITE_SPACE ('')
ITsemi ('')